app/src/main/res/values/strings.xml

```diff
@@ -416,9 +415,9 @@
     <string name="unit_settings_title">Units</string>
     <string name="no_chart_data_available">No data</string>
     <string name="remove_image">Remove Image</string>
-    <string name="program_completed">You have completed your program ! Do you want to close it ?</string>
-    <string name="database_migration">Database migration</string>
+    <string name="program_completed">Close completed program?</string>
+    <string name="database_migration">Move database</string>
```

Comment:
It is not really the database that is moving but rather the file that are referenced in the database that are moved. 
So it should probably just be "Move app files"